app/main.py

```diff
@@ -1,3 +1,49 @@
+def _convert_to_km(other: (int, float)) -> float:
```

Comment:
If you use such a helper it would make sense to have it as a method so it is bound to the class